changelog/changes/poetry-almost-mobile.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "from_http server=true` assertion failures"
+type: bugfix
+authors: raxyte
+pr: 5325
+---
+
+`from_http server=true` failed internal assertions and stopped the pipeline on
+receiving requests when `metadata_field` was specified.
```

Comment:
Some prepositions are missing here to make this difficult to read. Maybe "with" after "failed"?

Reply:
Added.